mass_mailing_partner/__openerp__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Link partners with mass-mailing",
-    "version": "8.0.1.0.0",
+    "version": "9.0.1.0.0",
     "author": "Antiun Ingeniería S.L., "
               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```

Comment:
Please switch both Antiun & Serv. Tecnol... to `Tecnativa`, that is the new company